test/pkg.jl

```diff
@@ -363,6 +363,29 @@ temp_pkg_dir() do project_path
         @test any(x -> startswith(x, manifest), keys(usage))
     end
 
+    @testset "atomic write_env_usage with $(Sys.CPU_THREADS) processes for 10 seconds" begin
+        tasks = Task[]
+        iobs = IOBuffer[]
+        Sys.CPU_THREADS == 1 && error("Cannot test for atomic usage log file interaction effectively with only Sys.CPU_THREADS=1")
+        for i in 1:Sys.CPU_THREADS
+            iob = IOBuffer()
+            t = @async run(pipeline(`$(Base.julia_cmd()) --project="$(pkgdir(Pkg))"
```

Comment:
Why would these subprocesses be precompiling Pkg?? Windows is failing because of atomic issues with `compilecache`, which shouldn't be needed here.. the master process should've precompiled Pkg.. 🤔